uniprot-search/src/main/resources/uniref/search-fields.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "field": "id",
+    "fieldValidRegex": "(UniRef100|UniRef90|UniRef50)\\w*",
+    "sortField": "id"
+  },
+  {
+    "field": "organism_sort",
```

Comment:
organism_sort is not a field in enum

Reply:
it's in the `Sort` nested enum of `UniRefField`. (So it is also a field in `schema.xml` for uniref)